### ISSUE TITLE
Montessori [ish] Counting Learning Material

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-cad-cam-cnc#65]
@Stephanymtr wants to **`make`** a counting learning material e.g: https://images.app.goo.gl/uFpQHAgsiBce2kLA7
![image](https://user-images.githubusercontent.com/194400/185590535-d7ecf509-288f-478a-9013-dbae5876a6a6.png)

and was considering making it using _paper_ 🥱  when I suggested using wood + laser ⚡ **`#pewpew`** 🤪 

I think we could make this [or a variation on this theme] 
in the workshop in **`T1h`** including design, cutting & assembly time. 💭

**Comment by nelsonic** [comment]
we are going to time-box building this now. 👨‍💻

**Comment by Stephanymtr** [comment]
We printed 1, 2 and 3. Here is what they look like: 

![20220819_115234.jpg](https://user-images.githubusercontent.com/91985721/185603657-87b69a55-b26f-473d-977a-72e685c5205b.jpg)
![20220819_115252.jpg](https://user-images.githubusercontent.com/91985721/185603693-4a1b4292-ec3c-4f1e-a9cd-80b8002f4cbd.jpg)![20220819_115327.jpg](https://user-images.githubusercontent.com/91985721/185603702-ac13f3b5-a13a-4880-897b-4d999e122331.jpg)![20220819_115343.jpg](https://user-images.githubusercontent.com/91985721/185603710-89eac9b0-a426-40f6-883e-85a27a0a4f4b.jpg)

**Comment by Stephanymtr** [comment]
![20220819_115543.jpg](https://user-images.githubusercontent.com/91985721/185603933-f93e6cdf-9430-42a1-ac9a-c6286e0a7927.jpg)

**Comment by nelsonic** [comment]
That was fun. Can’t wait for the _next_ mini-project. 👌🏻

**Comment by nelsonic** [comment]
@Stephanymtr have you had a chance to test this on our munchkin? 👶 🤞

**Comment by Stephanymtr** [comment]
@nelsonic Yes, I did. We did it twice, and he definitely enjoyed it. I noticed that when we did the activity (a month ago), three was still a bit too many to do at once. I layed out the counting cards on the floor and presented the activity to him. He then proceeded to do it but not in order. He placed 1 pompom on number 3, another on 1, then another on 2 and so on. The concept of numbers and quantity is new to him, so it'd be best to do one at time. Next time, I'd say to do the activity with just number one. I will plan to do this activity with him again this week and update this issue 👍

**Comment by nelsonic** [comment]
Good feedback. Look forward to more progress. 👌
